plugin.rb

```diff
@@ -81,33 +84,33 @@ class Engine < ::Rails::Engine
     reactions << like_reaction
   end
 
-  add_to_serializer(:post, :current_user_reactions) do
-    return [] unless scope.user.present?
-    reactions = object.reactions.map do |reaction|
+  add_to_serializer(:post, :current_user_reaction) do
+    return nil unless scope.user.present?
```

Comment:
for example this line here should have cause a test change

Reply:
This test is available in the code, i have fixed in the latest commit